src/workers/notifications/commentReply.ts

```diff
@@ -47,6 +45,7 @@ const worker: NotificationWorker = {
       .findBy({
         referenceId: In([comment.id, parent.id]),
         notificationType: In(commentReplyNotificationTypes),
+        status: NotificationPreferenceStatus.Muted,
```

Comment:
Hmm curious how we missed this?

Reply:
you can't really subscribe to comments, so no harm done